### PR TITLE
Migrate to Gradle (#1)

### PR DESCRIPTION
**Pull request by NyCodeGHG**
Removed pom.xml
Added gradle stuff and buildscript

### CODE REVIEW

**Comment by DRSchlaubi** [comment]
Yeah this should fix #1

**Comment by NyCodeGHG** [comment]
OHA D:

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.2-bin.zip
```
Comment (DRSchlaubi) [inline gradle/wrapper/gradle-wrapper.properties:3]: this should be all

**Review by DRSchlaubi — CHANGES_REQUESTED** [review]

**Comment by NyCodeGHG** [comment]
frechdachs

**Comment by NyCodeGHG** [comment]
The commit is not visible because @DerBanko closed the PR. I think if he got involved with Gradle, he would be fine. What does he have to lose?

**Comment by warriorzz** [comment]
maven

**Comment by DRSchlaubi** [comment]
Can someone provide CPR for maven?????
I think it's dying

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (DRSchlaubi) [reply gradle/wrapper/gradle-wrapper.properties:3]: I'll believe this comment: https://github.com/DerBanko/BoardGames/pull/2#issuecomment-774537783

**Pull request closed without merging**